setup.py

```diff
@@ -109,8 +113,8 @@
 setup_requires = ['numpy>=1.9']
 
 # pretty much needed
-install_requires = ['numpy>=1.9', 'qtpy>=1.1', 'setuptools>=1.0',
-                    'astropy>=1.0']
+install_requires = ['numpy>=1.13', 'qtpy>=1.1', 'setuptools>=1.0',
+                    'astropy>=3']
```

Comment:
Might as well? Is there a reason to drop PY2 support but retain compat with Astropy as old as v1? Numpy 1.13 bump goes with Astropy bump.